modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/content/Language.properties

```diff
@@ -12,6 +12,7 @@ relying-party-id=Relying Party ID
 relying-party-id-description=The RelyingParty ID must be equal to the origin's effective domain, or a registrable domain suffix of the origin's effective domain. For example an origin of https://login.example.com:1337 can only have rpID as one of the following: login.example.com or example.com. This is done in order to match the behavior of pervasively deployed ambient credentials (e.g., cookies,  [RFC6265])
 relying-party-name=Relying Party Name
 relying-party-name-description=Relying Party Name
+start=Start
```

Comment:
I have no idea if this is needed

Reply:
Check in `portal-impl/Language.properties`. If it isn't there, then it's needed.

Reply:
Thanks, after checking I can see it [here](https://github.com/liferay/liferay-portal/blob/9fac6b722b65cad46ea138de078f8cbaf97589ed/portal-impl/src/content/Language_en.properties#L4633) so I'll remove that